guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,352 @@
+---
+layout: guide
+title: The Payjoin Experience
+description: <>
+nav_order: 5
+parent: Case studies
+permalink: /guide/case-studies/payjoin-case-study/
```

Comment:
Could you please move this page to `/guide/case-studies/payjoin/`? No need to repeat `case study` twice in the URL. That also means moving files and folders around and updating various URLs in this PR.